yolo:
  actions:
    yolo:
      topic: yolov5_as/check_for_objects

  model:
    name: ultralytics/yolov5
    weight: best_yolov5s_600_f10_ev.pt
    exec: yolov5_detect.py

  local:
    yolo_path: yolov5/scripts
    weight_path: yolov5/weights
    image_path: yolov5/images
    image: det.jpg